api/server/httpstatus/status.go

```diff
@@ -21,7 +20,7 @@ type causer interface {
 // FromError retrieves status code from error message.
 func FromError(err error) int {
 	if err == nil {
-		log.G(context.TODO()).WithFields(logrus.Fields{"error": err}).Error("unexpected HTTP error handling")
+		log.G(context.TODO()).WithFields(log.Fields{"error": err}).Error("unexpected HTTP error handling")
```

Comment:
nit: This is equivalent to: 
```suggestion
		log.G(context.TODO()).WithError(err).Error("unexpected HTTP error handling")
```
which is a bit shorter 😄

Reply:
Good catch; will update.

While looking at it, I also noticed the `if err == nil` above it, but I guess it's intentional here to indicate "hey, we should've had an error here?"

Reply:
done 👍